ban/http/resources.py

```diff
@@ -296,6 +297,22 @@ class Group(WithHousenumbers):
 class Postcode(WithHousenumbers):
     model = models.PostCode
     order_by = [model.code, model.municipality]
+    param_whitelist = ['code']
+
+    def get_where_clause(self, req, qs):
+        for param in self.param_whitelist:
+            param_value = req.get_param_as_list(param)
```

Comment:
`values` instead of `param_value`: it's shorter while signicant, and plural says it's a multivalue.